Tests/Editor/SettingsAnalysisTests.cs

```diff
@@ -560,5 +560,45 @@ void ResetQualityLevelsValues(List<bool> values)
                 QualitySettings.streamingMipmapsActive = values[i];
             }
         }
+
+        [Test]
+#if !UNITY_2019_3_OR_NEWER
+        [Ignore("This requires the new Shader API")]
+#endif
+        public void SrpAssetSettingsAnalysis_SrpBatching_IsNotReportedOnceFixed()
+        {
+#if UNITY_2019_3_OR_NEWER
+            SrpAssetSettingsAnalyzer.SetSrpBatcherSetting(GraphicsSettings.defaultRenderPipeline);
```

Comment:
This looks a bit as if the test sets two settings, and doesn't reset them later to the original.

So just to be sure, the test should make sure that after the test the original serialized settings are the same.

Reply:
Right, I've fixed the test.